developer/src/kmc-convert/test/keylayout-to-kmn-converter.tests.ts

```diff
@@ -839,31 +640,30 @@ describe('KeylayoutToKmnConverter', function () {
     ].forEach(function (values) {
       it(("get_KeyActionOutput_array__From__ActionStateOutput_array(['" + values[0][0][0] + "', '" + values[0][0][1] + "', '" + values[0][0][2] + "'])").padEnd(73, " ") + ' should return an array of objects', async function () {
         const result = sut.get_KeyActionOutput_array__From__ActionStateOutput_array(read, values[0]);
-        assert.equal(JSON.stringify(result), JSON.stringify(values[1]));
+        await doesNotReject(async () => JSON.stringify(result) === JSON.stringify(values[1]));
```

Comment:
I would expect this would still have asserts - you can do `import * as assert from 'node:assert'` and then this line wouldn't need to change